src/components/views/auth/RegistrationForm.js

```diff
@@ -91,14 +91,25 @@ module.exports = React.createClass({
 
         const self = this;
         if (this.state.email == '') {
+            const haveIs = Boolean(this.props.serverConfig.isUrl);
+
+            let desc;
+            if (haveIs) {
+                desc = _t(
+                    "If you don't specify an email address, you won't be able to reset your password. " +
+                    "Are you sure?",
+                );
+            } else {
+                desc = _t(
+                    "No Identity Server is configured so you cannot add add an email address in order to " +
```

Comment:
... unless the HS handles password resets (as is the trend now)

Reply:
Yeah, but this is the thing where we don't know if the HS requires an ID server to do password reset or not :(

Reply:
i think we should assume the HS does the password reset for security. If someone is running a setup where the IS has to do it, the onus is on them to ensure an IS is set.

Reply:
> you cannot add add